components/create_comment/create_comment.jsx

```diff
@@ -184,6 +184,16 @@ class CreateComment extends React.PureComponent {
          * The last time, if any, when the selected post changed. Will be 0 if no post selected.
          */
         selectedPostFocussedAt: PropTypes.number.isRequired,
+
+        /**
+         * Update preview for textbox
+         */
+        updatePreview: PropTypes.func.isRequired,
+
+        /**
+         * Should preview be showed
+         */
+        showPreview: PropTypes.bool.isRequired,
```

Comment:
`showPreview`, while what we used before, also has the connotation of being something that is done (i.e. "show the preview") vs. something that is. Might we also take the opportunity to transition to using `isShowingPreview`  or some such?